docs/bc-registries-audit-notification-workflow.md

```diff
@@ -0,0 +1,82 @@
+
+There are bunch of services always running. So if those services are interrupted or Internet connectivity is down, that can affect the synchorization of data for a little while.
+
+## Digital Trust Monitoring Services OCP Environments
+
+Links to the Deployment Configurations Console can be found here; [Administrator - Deployment Configurations Console](./digital-trust-monitoring-services-environments)
+
+## BC Registries Agent Configurations `./manage` script
+
+General information regarding the bcgov/bc registries-agent-configurations openshift `./manage` script can be found here; [bcgov/bc-registries-agent-configurationsopenshift `./manage` script](./bc-registries-agent-configurations-manage-script.md)
+
+Specifics regarding the commands referenced in this workflow can be found here:
+- [`requeueOrganization` command](./bc-registries-agent-configurations-manage-script.md#requeueOrganization-command)
+- [`queueOrganization` command](./bc-registries-agent-configurations-manage-script.md#queueOrganization-command)
+
+
+## What causes the alert condition to surface?
+In general we see these notifications when the BC Registries Audit scripts detect any discrepancy between the data in the BC Registries database(s) and the data in the OrgBook.
+
+For Example: Corp Type mis-match for: BC1057752; BC Reg: "BEN", OrgBook: "BC"
+
+### What steps needs to be taken to resolve this error?
+
+In order to solve the alert above.
+
+Confirm the alert by browsing to ""Orgbook.gov.bc.ca"" to verify 
+the alert received in BC Registries Audit Notification is not false positive by ensuring the data of Orgbook matches with BC Registries;
+
+Once the alert is confimed
+
+Log into the affected OCP environment;
+
+
+```
+./manage -p bc -e prod deleteTopic BC1057752
+./manage -e prod requeueOrganization 1057752
+```
+
+In this example, the corp type mis-match represents companies Status mis-matching in OrgBook, and must be queued from BC Reg
+
+The first line represents the status of company in OrgBook is incorrect and must be deleted and re-processed.
+
+The commands should be run in the order listed, however, if, for some reason, you need to run the commands in a batch, the `orgbook-configurations` scripts should always be run first.
+
+Commands in the following forms should be run using the openshift `./manage` scripts from [orgbook-configurations](https://github.com/bcgov/orgbook-configurations):
+- `./manage -p bc -e <env> deleteTopic <business_number>`
+
+Commands in the following forms should be run using the openshift `./manage` scripts from [von-bc-registries-agent-configurations](https://github.com/bcgov/von-bc-registries-agent-configurations):
+- `./manage -e <env> queueOrganization <business_number>`
+- `./manage -e <env> requeueOrganization <business_number>`
```

Comment:
These instructions should mirror the flow depicted in the decision tree and provide additional details for each step, that way the diagram can act as a point form summary of the workflow.